src/pipeline-server/Dockerfile.pipeline-server

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (C) 2024 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+FROM intel/dlstreamer-pipeline-server:2022.2.0-ubuntu20-gpu419.40
```

Comment:
do they have later version? or this is working version for now

Reply:
This is the latest public release we have access to